src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -94,7 +94,7 @@ export const DesktopDropdown = ({
           isOpen={isOpen}
           optionsContainerMaxHeight={optionsContainerMaxHeight}
           role="menu"
-          aria-activedescendant={value}
+          aria-activedescendant={value as string}
```

Comment:
Can we remove these two `as string` casts now that we've widened the `value` type? Especially for `aria-activedescendant`, which doesn't look like it needs casting:

```tsx
aria-activedescendant: string | number | null | undefined
```